src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -334,6 +345,7 @@ const DashboardEditor = ({
             selectedBreakpointIndex={selectedBreakpointIndex}
             setSelectedBreakpointIndex={setSelectedBreakpointIndex}
             breakpointSwitcher={breakpointSwitcher}
+            isLoading={isLoading}
```

Comment:
Isn't this always `false` based on the ternary this is contained within?

Reply:
Oops, meant to remove that